src/configure/mod.rs

```diff
@@ -96,11 +97,12 @@ impl Context {
 
         match success {
             Ok(_) => {
+                self.set_config(Config::NotifyAboutWrongPw, Some("1"))
+                    .await?;
                 progress!(self, 1000);
                 Ok(())
             }
             Err(err) => {
-                error!(self, "Configure Failed: {}", err);
```

Comment:
hm, does this really work? with progress!() the event handler is called with and error and no error was reported.